libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -625,16 +625,16 @@ void AC_PosControl::update_xy_controller()
     }
     _last_update_xy_us = AP_HAL::micros64();
 
-    float ekfGndSpdLimit, ekfNavVelGainScaler;
-    AP::ahrs().getEkfControlLimits(ekfGndSpdLimit, ekfNavVelGainScaler);
+    float ekfGndSpdLimit, ekfControlScaleXY;
+    AP::ahrs().getEkfControlLimits(ekfGndSpdLimit, ekfControlScaleXY);
```

Comment:
```suggestion
    float ahrsGndSpdLimit, ahrsControlScaleXY;
    AP::ahrs().getEkfControlLimits(ahrsGndSpdLimit, ahrsControlScaleXY);
```

If we're going to "fix" the naming then let's fix it.

Reply:
done